rfcs/react-components/convergence/portal-compat.md

```diff
@@ -0,0 +1,166 @@
+# RFC: Portal compatibility
+
+[@layershifter](https://github.com/layershifter)
+
+## Summary
+
+This RFC proposes introduce compatibility between components in different Fluent UI versions that use [`ReactDOM.createPortal()`](https://reactjs.org/docs/portals.html) via:
+
+- usage of same values for `z-index`
+- usage of separate React context to assign CSS class that handles CSS variables from Fluent UI React v9
+
+## Problem statement
+
+Once components from Fluent UI React v9 (`@fluentui/react-components`) are put inside popovers & dialogs from Fluent UI React v8 (`@fluentui/react`) or Fluent UI React v0 (`@fluentui/react-northstar`) they don't have proper styles 💥
```

Comment:
Let's explicitly mention that customers on v7 will not get compat support by us ?

Reply:
Done